Cannot start in Windows.

After I followed the steps in the README.md to install this extension, I couldn't start the service.

Error: Context server request timeout.

![Image](https://github.com/user-attachments/assets/1711781d-bdd1-4f14-a4c7-b584efadc276)

I think it errors because it can't find index.js, path is not composed correctly on Windows, from logs it tries to load
[context_server::client] context server stderr: Error: Cannot find module 'C:\C:\Users\USER\AppData\Local\Zed\extensions\work\mcp-server-context7\node_modules\@upstash\context7-mcp\dist\index.js'
Obviously, 'C:\C:\' is not a thing.

> I think it errors because it can't find index.js, path is not composed correctly on Windows, from logs it tries to load [context_server::client] context server stderr: Error: Cannot find module 'C:\C:\Users\USER\AppData\Local\Zed\extensions\work\mcp-server-context7\node_modules@upstash\context7-mcp\dist\index.js' Obviously, 'C:\C:' is not a thing.

you can give file path manually like as me.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -10,6 +10,6 @@ path = "src/mcp_server_context7.rs"
 crate-type = ["cdylib"]
 
 [dependencies]
-zed_extension_api = "0.5.0"
+zed_extension_api = "0.7.0"
 serde = "1.0"
 schemars = "0.8"

--- a/extension.toml
+++ b/extension.toml
@@ -1,7 +1,7 @@
 id = "mcp-server-context7"
 name = "Context7 MCP Server"
 description = "Model Context Protocol Server for Context7"
-version = "0.0.4"
+version = "0.0.5"
 schema_version = 1
 authors = ["Akbxr <hi@akbxr.com>"]
 repository = "https://github.com/akbxr/zed-mcp-server-context7"

